doc/sources.rst

```diff
@@ -253,6 +254,85 @@ Full example::
       transparent: true
 
 
+.. _arcgis_label:
+
+ArcGIS REST API
+"""
+
+Use the type ``arcgis`` to for ArcGIS servers.
+
+``req``
+^^^^^^^
+
+This describes the ArcGIS source. The only required option is ``url``. You need to set ``transparent`` to ``true`` if you want to use this source as an overlay.
+::
+
+  req:
+    url: http://example.org/ArcGIS/rest/services/Imagery/MapService
+    layers: show: 0,1
+    transparent: true
+
+``coverage``
+^^^^^^^^^^^^
```

Comment:
Can you refer to the WMS source documentation for the redundant parts? The MapServer docs starts with "This source is based on the WMS source and most options apply to the Mapserver source too." You can list (and link) the specific options that are identical if you like (i.e coverage, supported_srs), but please avoid duplication :)